test/snippet/alphabet/structure/wuss_pseudoknot_id.cpp

```diff
@@ -3,11 +3,11 @@
 
 int main()
 {
-    using seqan3::operator""_wuss51;
+    using namespace seqan3::literals;
 
     auto pk_opt = '.'_wuss51.pseudoknot_id();                       // std::optional -> false
     pk_opt = seqan3::pseudoknot_id('{'_wuss51);                     // std::optional -> true: 3
 
     if (pk_opt)
-        seqan3::debug_stream << *pk_opt;                            // 3
+        seqan3::debug_stream << *pk_opt << '\n';                    // 3
```

Comment:
Why is the newline not needed?

Reply:
It's never needed, it's just annoying if you run the snippet and it doesn't print a newline